requirements-base.txt

```diff
@@ -1,3 +1,4 @@
+batching-kafka-consumer==0.0.3
```

Comment:
Hm, not sure if this belongs in base or what. We'll need it in production, but I guess the Kafka client is giving people trouble in dev.

Reply:
I went ahead and moved this to optional, but it's worth noting that you can't run the test suite (on `master`) without `confluent-kafka` anyway.